src/wazuh_db/wdb.h

```diff
@@ -1270,6 +1271,8 @@ void wdbi_update_attempt(wdb_t * wdb, wdb_component_t component, long timestamp,
 
 void wdbi_update_completion(wdb_t * wdb, wdb_component_t component, long timestamp, os_sha1 checksum);
 
+void wdbi_set_last_completion_only(wdb_t * wdb, wdb_component_t component, long timestamp);
```

Comment:
I think **wdbi_set_last_completion()** will be enough to represent what this method does.

Reply:
Fixed!